concurrency/runner.go

```diff
@@ -62,45 +63,53 @@ func ForEachUser(ctx context.Context, userIDs []string, concurrency int, userFun
 
 // ForEach runs the provided jobFunc for each job up to concurrency concurrent workers.
 // The execution breaks on first error encountered.
+// Deprecated.
+// Use ForEachJob instead.
```

Comment:
```suggestion
// 
// Deprecated: use ForEachJob instead.
```

(based on https://go.dev/blog/godoc: "To signal that an identifier should not be used, add a paragraph to its doc comment that begins with “Deprecated:” followed by some information about the deprecation.)

Reply:
Oh it has to be a colon :facepalm: I was trying and this was the only way my IDE understood it.